### PR TITLE
Extracted spring, supported micronaut

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: eventuate-messaging-kafka-consumer-micronaut/src/main/java/io/eventuate/messaging/kafka/consumer/micronaut/MessageConsumerKafkaFactory.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+import io.eventuate.messaging.kafka.basic.consumer.EventuateKafkaConsumerConfigurationProperties;
+import io.eventuate.messaging.kafka.common.EventuateKafkaConfigurationProperties;
+import io.eventuate.messaging.kafka.consumer.MessageConsumerKafkaImpl;
+import io.micronaut.context.annotation.Factory;
+import io.micronaut.context.annotation.Requires;
+
+import javax.inject.Singleton;
+
+//@Requires(property = "micronaut.eventuate.message.consumer.kafka.factory", value = "true")
```
Comment (cer) [inline eventuate-messaging-kafka-consumer-micronaut/src/main/java/io/eventuate/messaging/kafka/consumer/micronaut/MessageConsumerKafkaFactory.java]: Please remove

File: gradle.properties [hunk 2]
```diff
@@ -8,6 +8,6 @@ eventuateMavenRepoUrl=https://dl.bintray.com/eventuateio-oss/eventuate-maven-rel
 springBootVersion=2.1.1.RELEASE
 
 eventuateUtilVersion=0.1.0.RELEASE
-eventuateMessagingPartitionManagementVersion=0.2.0.RELEASE
+eventuateMessagingPartitionManagementVersion=0.1.0-SNAPSHOT
```
Comment (cer) [inline gradle.properties]: There is no PR for eventuate-common-messaging.
Also, eventuateMessagingPartitionManagementVersion should be renamed to eventuateCommonMessagingVersion

File: eventuate-messaging-kafka-consumer-micronaut/src/main/java/io/eventuate/messaging/kafka/consumer/micronaut/MessageConsumerKafkaFactory.java (hunk 1 above)
Comment (dartartem) [reply eventuate-messaging-kafka-consumer-micronaut/src/main/java/io/eventuate/messaging/kafka/consumer/micronaut/MessageConsumerKafkaFactory.java]: ok

File: gradle.properties (hunk 2 above)
Comment (dartartem) [reply gradle.properties]: There was not PR, because it required released common libraries.
Could you please release it?
I will rename.

**Comment by cer** [comment]
merged into branch wip-micronaut

**Pull request closed without merging**